platform/pulp/tasking/tasks.py

```diff
@@ -64,6 +64,12 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
                           will ensure that no other tasks that want that same reservation will run
                           concurrently with yours.
     :type  resource_id:   basestring
+    :param inner_args:    The positional arguments to pass on to the task.
+    :type  inner_args:    tuple
+    :param inner_kwargs:  The keyword arguments to pass on to the task.
+    :type  inner_kwargs:  dict
+    :param **options:     For all options accepted by apply_async please visit: http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.apply_async  #NOQA
```

Comment:
Since this is a positional argument in the function signature this docstring should be s/**//.